### ISSUE TITLE
ERROR: Cannot find import .

### ISSUE THREAD

**Issue by FrimJo** [benawad/destiny#132]
Hi guys and gals, I can't seem to find any similar issue here.

I have this project and file structure as shown below in the debug output, and I get this weird error as shown at the end of debug output. I've seen similar problems but then the error points towards a specific file, for me it just points to `.`

```bach
npx destiny "src/**/*.*" --debug 
DEBUG: +0ms version: 0.6.1
DEBUG: +1ms config used:
  {
    help: false,
    include: [ 'src/**/*.*' ],
    version: false,
    write: false,
    avoidSingleFile: false,
    debug: true
  } 
  
DEBUG: +21ms glob matches for "src/**/*.*":
  [
    'src/api/use-authenticate-api.ts',
    'src/api/use-user-api.ts',
    'src/components/app/app.test.tsx',
    'src/components/app/app.tsx',
    'src/components/button/button-view.tsx',
    'src/components/display-user/display-user-container.tsx',
    'src/components/is-fetching-snackbar/is-fetching-snackbar-view.tsx',
    'src/components/router/components/private-route/private-route-container.tsx',
    'src/components/router/router-container.tsx',
    'src/components/spinner/spinner-style.ts',
    'src/components/spinner/spinner-view.tsx',
    'src/components/status-snackbar-error/status-snackbar-error-boundary.tsx',
    'src/components/status-snackbar-error/status-snackbar-error-view.tsx',
    'src/components/theme-provider/theme-provider-container.tsx',
    'src/components/theme-provider/theme-provider.test.tsx',
    'src/config/page-routes.ts',
    'src/config/variables.ts',
    'src/contexts/local-storage-context.ts',
    'src/index.tsx',
    'src/pages/account/pages/login-page/login-page-view.tsx',
    'src/pages/account/pages/login-page/utilities/use-login-form.ts',
    'src/pages/account/pages/register-page/register-page-view.tsx',
    'src/pages/account/pages/register-page/utilities/use-register-form.ts',
    'src/pages/account/pages/register-page/utilities/use-register.ts',
    'src/pages/not-found-page/not-found-page-view.tsx',
    'src/pages/private-page/private-page-view.tsx',
    'src/pages/start-page/components/user-page/user-page-container.tsx',
    'src/pages/start-page/components/user-page/user-page-view.tsx',
    'src/pages/start-page/components/user-page/utilities/use-user-form.ts',
    'src/pages/start-page/start-page-view.tsx',
    'src/pages/unauthorized-page/unauthorized-page-view.tsx',
    'src/react-app-env.d.ts',
    'src/service-worker.ts',
    'src/styles/colors.ts',
    'src/styles/global-styles.ts',
    'src/styles/index.css',
    'src/styles/theme.ts',
    'src/types/index.d.ts',
    'src/types/react-intl/index.d.ts',
    'src/types/react-query-devtools/index.d.ts',
    'src/utilities/build-context.test.tsx',
    'src/utilities/build-context.tsx',
    'src/utilities/history.ts',
    'src/utilities/react-router-hooks.tsx',
    'src/utilities/use-authentication.ts',
    'src/utilities/use-debounce.ts',
    'src/utilities/use-http-middleware.ts',
    'src/utilities/use-local-storage.ts',
    'src/utilities/use-logged-in-user.ts',
    'src/utilities/use-register-storage.ts',
    'src/utilities/use-token-storage.ts',
    'src/utilities/use-user.ts',
    'src/utilities/with-spinner.tsx'
  ] 
  
DEBUG: +1ms restructured map:
  {
    'src/**/*.*': [
      'src/api/use-authenticate-api.ts',
      'src/api/use-user-api.ts',
      'src/components/app/app.test.tsx',
      'src/components/app/app.tsx',
      'src/components/button/button-view.tsx',
      'src/components/display-user/display-user-container.tsx',
      'src/components/is-fetching-snackbar/is-fetching-snackbar-view.tsx',
      'src/components/router/components/private-route/private-route-container.tsx',
      'src/components/router/router-container.tsx',
      'src/components/spinner/spinner-style.ts',
      'src/components/spinner/spinner-view.tsx',
      'src/components/status-snackbar-error/status-snackbar-error-boundary.tsx',
      'src/components/status-snackbar-error/status-snackbar-error-view.tsx',
      'src/components/theme-provider/theme-provider-container.tsx',
      'src/components/theme-provider/theme-provider.test.tsx',
      'src/config/page-routes.ts',
      'src/config/variables.ts',
      'src/contexts/local-storage-context.ts',
      'src/index.tsx',
      'src/pages/account/pages/login-page/login-page-view.tsx',
      'src/pages/account/pages/login-page/utilities/use-login-form.ts',
      'src/pages/account/pages/register-page/register-page-view.tsx',
      'src/pages/account/pages/register-page/utilities/use-register-form.ts',
      'src/pages/account/pages/register-page/utilities/use-register.ts',
      'src/pages/not-found-page/not-found-page-view.tsx',
      'src/pages/private-page/private-page-view.tsx',
      'src/pages/start-page/components/user-page/user-page-container.tsx',
      'src/pages/start-page/components/user-page/user-page-view.tsx',
      'src/pages/start-page/components/user-page/utilities/use-user-form.ts',
      'src/pages/start-page/start-page-view.tsx',
      'src/pages/unauthorized-page/unauthorized-page-view.tsx',
      'src/react-app-env.d.ts',
      'src/service-worker.ts',
      'src/styles/colors.ts',
      'src/styles/global-styles.ts',
      'src/styles/index.css',
      'src/styles/theme.ts',
      'src/types/index.d.ts',
      'src/types/react-intl/index.d.ts',
      'src/types/react-query-devtools/index.d.ts',
      'src/utilities/build-context.test.tsx',
      'src/utilities/build-context.tsx',
      'src/utilities/history.ts',
      'src/utilities/react-router-hooks.tsx',
      'src/utilities/use-authentication.ts',
      'src/utilities/use-debounce.ts',
      'src/utilities/use-http-middleware.ts',
      'src/utilities/use-local-storage.ts',
      'src/utilities/use-logged-in-user.ts',
      'src/utilities/use-register-storage.ts',
      'src/utilities/use-token-storage.ts',
      'src/utilities/use-user.ts',
      'src/utilities/with-spinner.tsx'
    ]
  } 

INFO: Generating tree for: src/**/*.*
ERROR: Cannot find import .
If you think this is a bug, you can report it: https://github.com/benawad/destiny/issues
DEBUG: +414ms exiting
```

**Comment by benawad** [comment]
Do you have an import that is just `.`

**Comment by AnatoleLucet** [comment]
@FrimJo We've just shipped a new version ('0.6.2') which should tell you in which file Destiny is unable to resolve your import.
Can you please try it again with this new version and share the result with us?

**Comment by FrimJo** [comment]
@benawad Sorry for taking up your time. It was one of my tests using `import … from '.'` which did not resolve, didn't notice because I just removed all my barrel index files from the project and didn't notice my tests had problems.
